oci/private/tarball.bzl

```diff
@@ -71,12 +74,23 @@ def _tarball_impl(ctx):
     ctx.actions.run(
         executable = util.maybe_wrap_launcher_for_windows(ctx, executable),
         inputs = [image, repo_tags, executable],
-        outputs = [tarball],
+        outputs = [mtree],
         tools = [yq_bin],
         mnemonic = "OCITarball",
```

Comment:
I would move this mnemonic and progress message to the new step (that calls `bsdtar.tarinfo.binary`) as that is the part that is effectively creating the Tarball. This one could be named differently.

Reply:
I had to revert to a single action, PTAL